static/js/components/Group.jsx

```diff
@@ -1,18 +1,129 @@
 import React, { useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router-dom";
 import PropTypes from "prop-types";
-import { Link, useParams } from "react-router-dom";
+
 import { useDispatch, useSelector } from "react-redux";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import DeleteIcon from "@material-ui/icons/Delete";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemSecondaryAction from "@material-ui/core/ListItemSecondaryAction";
+import ListItemText from "@material-ui/core/ListItemText";
+import IconButton from "@material-ui/core/IconButton";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+
+import Accordion from "@material-ui/core/Accordion";
+import AccordionSummary from "@material-ui/core/AccordionSummary";
+import AccordionDetails from "@material-ui/core/AccordionDetails";
+import Typography from "@material-ui/core/Typography";
+
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import TextField from "@material-ui/core/TextField";
+import OpenInNewIcon from "@material-ui/icons/OpenInNew";
+
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import Divider from "@material-ui/core/Divider";
+import Chip from "@material-ui/core/Chip";
+import CircularProgress from "@material-ui/core/CircularProgress";
+
+import { useForm, Controller } from "react-hook-form";
+import { showNotification } from "baselayer/components/Notifications";
 
 import * as groupActions from "../ducks/group";
 import * as groupsActions from "../ducks/groups";
+import * as streamsActions from "../ducks/streams";
+import * as filterActions from "../ducks/filter";
 import NewGroupUserForm from "./NewGroupUserForm";
 
-import styles from "./Group.css";
+const useStyles = makeStyles((theme) => ({
+  padding_bottom: {
+    paddingBottom: "2em",
+  },
+  paper: {
+    width: "100%",
+    padding: theme.spacing(1),
```

Comment:
@acrellin Do you know if this is absolute or relative spacing?  I would guess absolute.